### PR TITLE
try to improve make.sh with tabs/spaces situation

### PR DESCRIPTION
**Pull request by eklitzke**
The `make.sh` script is not at all consistent on hard tabs vs. 8-space tabs.

The diff to make it use hard tabs is smaller, but I don't care. I actually slightly prefer the other way (8-space softtabs)

### CODE REVIEW

**Comment by aquynh** [comment]
unfortunately, this does not pass Travis. could you take a look? 

https://travis-ci.org/aquynh/capstone/builds/122447032

thanks

**Comment by 07151129** [comment]
Bash is (usually) `/bin/bash` without `/usr/` prefix. Alternatively, bring usage of `env` back.

**Comment by aquynh** [comment]
do you still use /bin/bash?

**Comment by aquynh** [comment]
@eklitzke, how about using /bin/sh rather than /bin/bash?

File: make.sh [hunk 1]
```diff
@@ -1,19 +1,19 @@
-#!/usr/bin/env bash
-
+#!/bin/bash
```
Comment (radare) [inline make.sh]: let's use /bin/sh, no need for bashisms here

File: make.sh [hunk 2]
```diff
@@ -24,40 +24,40 @@ function build_iOS {
...
-          ;;
-    esac
-
-    TOOLCHAIN="$NDK/toolchains/$CROSS$GCCVER/prebuilt/$HOSTOS-$HOSTARCH"
-    PLATFORM="$NDK/platforms/$APILEVEL/arch-$TARGARCH"
+	if [ -z "$NDK" ]; then
+		echo "ERROR! Please set \$NDK to point at your Android NDK directory."
+		exit 1
+	fi
+	HOSTOS="$(uname -s | tr LD ld)"
```
Comment (radare) [inline make.sh]: why LD?

Comment (radare) [inline make.sh]: i assume because of Linux and Darwin.. but thats a weird assumption when we can just lowercase the whole thing

**Comment by radare** [comment]
please rebase and fix the comments. ok to merge after this

**Pull request closed without merging**